examples/Example_OLR_Tracking_satellite/Example_OLR_Tracking_satellite.ipynb

```diff
@@ -79,15 +79,20 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# # Download the data: This only has to be done once for all tobac examples and can take a while\n",
-    "# file_path='https://zenodo.org/record/3195910/files/climate-processes/tobac_example_data-v1.0.1.zip'\n",
-    "# tempfile='temp.zip'\n",
-    "# print('start downloading data')\n",
-    "# request=urllib.request.urlretrieve(file_path,tempfile)\n",
-    "# print('start extracting data')\n",
-    "# zf = zipfile.ZipFile(tempfile)\n",
-    "# zf.extractall(data_out)\n",
-    "# print('example data saved in')"
+    "# Download the data: This only has to be done once for all tobac examples and can take a while\n",
+    "data_path=os.path.join(data_out,'*','data','Example_input_OLR_satellite.nc')\n",
```

Comment:
Not sure how important this is, but since you are updating this part now anyhow, maybe we could also replace the `os.path` functions by using [pathlib](https://docs.python.org/3/library/pathlib.html) to go with the flow?

Reply:
I'll have to look into it, as I haven't used pathlib before. Maybe it's something for a future update?

Reply:
OK, sounds good! Lets save this for another time and consider this PR approved :) 

Just for info: My understanding is that in general it could be advantageous to migrate to `pathlib,` since it is a bit easier to read and to use. For this short code snippet maybe not so relevant, but there a lot of operations that you can directly apply on `pathlib.Path` objects, whereas you would need additional libraries with `os.path`. It is also quite straight forward to convert, your code would maybe look something like: 

```suggestion

from pathlib import Path 
data_path=Path(data_out)
data_file = list(data_path.rglob('*/data/Example_input_OLR_model.nc'))
if len(data_file) == 0 :
    file_path='https://zenodo.org/record/3195910/files/climate-processes/tobac_example_data-v1.0.1.zip'
    tempfile= Path('temp.zip')
    print('start downloading data')
    request=urllib.request.urlretrieve(file_path,tempfile)
    print('start extracting data')
    shutil.unpack_archive(str(tempfile),str(data_out))
    tempfile.unlink()
    print('data extracted')
    data_file = list(data_path.glob('*'))
```

Reply:
Thanks for the example, I went through the example notebooks to update them to use pathlib, and it actually meant that we can replace os and glob completely as imports